nuxeo-compound-documents/src/main/java/org/nuxeo/compound/documents/CompoundDocumentServiceImpl.java

```diff
@@ -108,12 +122,13 @@ protected void createEntry(DocumentModel compoundDoc, ZipFile zip, ZipEntry entr
         } else {
             var blob = new ZipEntryBlob(zip, entry);
             blob.setFilename(entryDocName);
-            FileImporterContext ctx = FileImporterContext.builder(session, blob, parentDocPath).build();
+            FileImporterContext ctx = FileImporterContext.builder(compoundDoc.getCoreSession(), blob, parentDocPath)
```

Comment:
I would have passed as a param `createEntry` method from the caller `createCompoundDocument` and reused it here.